### ISSUE TITLE
Every field value is a URL

### ISSUE THREAD

**Issue by mskyttner** [DINA-Web/collections-ui#69]
And can be clicked. To get to the data. Idea from GN.

**Comment by caalle** [comment]
Yes, this is relevant when having relationships. Eg. clicking the collection name of a collection object would take you to a collection view page displaying information about the collection and all collection objects registered in that collection.
